packages/devtools_shared/lib/src/server/server_api.dart

```diff
@@ -94,17 +92,11 @@ class ServerApi {
         return _encodeResponse(true, api: api);
       case apiGetDevToolsFirstRun:
         // Has DevTools been run first time? To bring up analytics dialog.
-        //
-        // Additionally, package:unified_analytics will show a message if it
-        // is the first run with the package or the consent message version has
-        // been updated
-        final isFirstRun =
-            _devToolsUsage.isFirstRun || analytics.shouldShowMessage;
+        final isFirstRun = _devToolsUsage.isFirstRun;
         return _encodeResponse(isFirstRun, api: api);
```

Comment:
Will we ever get in a state now where the legacy and new analytics systems are in disagreements with each other and possibly causing errors?

Reply:
There should not be any logic changes WRT analytics in this PR; only changes to where the unified analytics logic is accessed (by proxy of DTD now). If you see a case where there is a logic change, this is not intentional so please flag it.